DAO.sol

```diff
@@ -40,14 +40,16 @@ contract DAOInterface {
     // Period after which a proposal can be closed
     // (used in the case `executeProposal` fails because it throws)
     uint constant executeProposalPeriod = 5 days;
-
+    // Denotes the maximim proposal deposit that can be given. It is given as
+    // a fraction of total Ether spent plus balance of the DAO
+    uint constant maxDepositDivisor = 100;
     // Proposals to spend the DAO's ether or to choose a new Curator
```

Comment:
at least 1 space between the function constants and the rest, it helps visually when reading the contract.